contrib/mw-to-git/git-remote-mediawiki.perl

```diff
@@ -73,6 +73,11 @@
 chomp($export_media);
 $export_media = !($export_media eq 'false');
 
+# Change files extension for external tool compatibility
+my $file_ext = run_git("config --get --bool remote.${remotename}.fileextension");
```

Comment:
Why `--bool`? I think it's a cut-and-paste error from elsewhere, but it shouldn't be there.